.ado/templates/upload-build-logs.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
+  buildLogDirectory: $(Build.BinariesDirectory)\$(BuildPlatform)\$(BuildConfiguration)\BuildLogs
+
+steps:
+  - task: PublishBuildArtifacts@1
+    displayName: Upload build logs
+    condition:  succeededOrFailed()
+    inputs:
+      pathtoPublish: '${{ parameters.buildLogDirectory }}'
```

Comment:
>buildLogDirectory [](start = 37, length = 17)

Does it matter that 'buildLogDirectory' here is capitalized differently than in the original task?

Reply:
no, parameters is a local concept.